lib/python/fidasim/preprocessing.py

```diff
@@ -174,7 +174,16 @@ def check_inputs(inputs, use_abs_path=True):
               'calc_birth': zero_int,
               'calc_fida_wght': zero_int,
               'calc_npa_wght': zero_int,
-              'calc_neutron': zero_int}
+              'calc_neutron': zero_int,
+              'cell_split': zero_int}
```

Comment:
small thing but could you call it `split_cell` so we follow the `verb_noun` naming scheme

Reply:
You could also just call it `adaptive` as that name is a bit more intuitive.

Reply:
Right now the `adaptive` marker only sets the cell to split based on a gradient in the electron density, is there a full list of all the parameters that a user could choose from aside from dene?

Reply:
@hayashiw, what do you exactly mean by `adaptive` marker?

Reply:
@alvin-garcia  I mean the `cell_split` marker, I'll change it to `adaptive` and have it so that a value of 0 turns it off, 1 turns it on for the default electron density, and higher values could correspond to other parameters.